marketplace/eqp/v1/users.md

```diff
@@ -38,17 +40,17 @@ curl -X GET \
 ```json
 {
         "mage_id": "MAG123456789",
-        "first_name": “Chuck”,
-        "last_name": “Norris”,
+        "first_name": "Chuck",
+        "last_name": "Norris",
```

Comment:
Let's change the example so that we're not using a celebrity name. Also, the value of the `email` attribute is missing the first quotation mark.